### ISSUE TITLE
compilation issue on ubutu18.10 with gcc 7.3

### ISSUE THREAD

**Issue by rafiw** [CLIUtils/CLI11#211]
```
#make V=1
g++ -Wextra -Wall -Wno-unknown-pragmas -Werror -ffunction-sections -fdata-sections -pipe -Wno-overloaded-virtual -Wmissing-include-dirs -Wnon-virtual-dtor -Woverloaded-virtual -Wunused-local-typedefs -Wvla -Werror -g -O2 -O2  -MT 
CLI11.hpp: In function 'size_t CLI::detail::escape_detect(std::__cxx11::string&, size_t)':
CLI11.hpp:448:52: error: ?: using integer constants in boolean context, the expression will always evaluate to 'true' [-Werror=int-in-bool-context]
             if(str[astart] == (str[offset] == '=') ? '-' : '/')

```

**Comment by phlptp** [comment]
think that was fixed in #201

**Comment by henryiii** [comment]
That should be fixed both in master and 1.7.1.
